lib/mail/utilities.rb

```diff
@@ -221,5 +221,24 @@ def map_with_index( enum, &block )
 
     end
 
+    # Returns true if the object is considered blank.
+    # A blank includes things like '', '   ', nil,
+    # and arrays and hashes that have nothing in them.
+    # 
+    # This logic is mostly shared with ActiveSupport's blank?
+    def self.blank?(string)
+      if string.is_a?(NilClass)
+        true
+      elsif string.is_a?(String)
```

Comment:
Will want to use `kind_of?` rather than `is_a?` on these to catch subclasses. Or use a `case` statement which'll use `#===` comparison.